index.html

```diff
@@ -178,15 +178,15 @@ <h1 class="about-title">About Myself</h1>
                         </div>
                         <ul class="languages">
                             <li class="programming-lang">
-                                <img src="images/javascrip.svg" alt="">
```

Comment:
:white_check_mark: Image text alternatives** (img alt="alt text") (We added appropiate alt texts to every image)